README.md

```diff
@@ -71,6 +71,10 @@ for c in child_nodes(xroot)  # c is an instance of XMLNode
         println(name(e))
     end
 end
+
+free(xdoc) # if the remainder of the script does not use the document or any of
+           # its children, you can call free here to deallocate the memory
+           # it will only get deallocated by calling free or by exiting julia
 ```
```

Comment:
Thanks for the contribution!

Given the size of the comment, it should probably be a block comment above `free(xdoc)`, rather than an extended line comment.

I realize that all of the other comments here have no capitalization or punctuation, but also consider, e.g., adding a semicolon, at the end of the second line, to give some visual separation between it and the sentence on the third line.

I'm not a regular contributor here, so anyone that is a regular contributor should feel free to override these suggestions.  (However, if no one else responds, I'll merge the PR with those changes.)

Reply:
Makes sense to me.